performance.lic

```diff
@@ -20,12 +20,10 @@ class Performance
 
     @settings = get_settings
     @song_list = get_data('perform').perform_options
-    @skip_clean = args.noclean
-
-    UserVars.song = @song_list.first.first unless UserVars.song
-    fput('release ecry') if DRSpells.active_spells["Eillie's Cry"].to_i > 0
+    skip_clean = args.noclean
 
     instrument = @settings.instrument
+    is_instrument_worn = instrument.nil?
```

Comment:
❓ Could you elaborate on how `nil?` is the right way to determine if an instrument should be worn?

Reply:
There might be a better way to do this, but -- settings.instrument is the setting for if you have an instrument you pull out of your bags.  Not zills/cowbell, in other words.  instrument is nil in base.yaml, and most people will never set it, but if it's set to not nil then performance does this:

```
    if instrument
      bput("get my #{instrument}", 'You get')
      bput("remove my #{hand_armor}", 'You pull off', 'Remove what')
      bput("stow my #{hand_armor}", 'You put', 'Stow what')
    end
```

Then before dying it will put 'instrument' away.  Otherwise it just plays, confident you have a worn instrument on.